index.html

```diff
@@ -128,6 +128,11 @@
 <pre><code>./run-tests.sh</code></pre>
 
 <h2>
+<a name="api-sparkr" class"anchor" href="#api-sparkr"><span class="octicon octicon-link"></span></a>SparkR package API documentation</h2>
+
+<a href="rdocs/00Index.html">Documentation for package ‘SparkR’</a>
```

Comment:
ah sorry. I couldn't see that in email. updated, great point. I didn't see the index.html